components/application-operator/charts/application/templates/role-binding.yaml

```diff
@@ -88,7 +88,10 @@ metadata:
     heritage: {{ .Release.Service }}
 rules:
 - apiGroups: ["*"]
-  resources: ["namespaces", "subscriptions", "triggers"]
```

Comment:
you dropped `subscriptions` deliberately?

Reply:
yes, it's not used anymore